Gemfile.lock

```diff
@@ -273,7 +273,7 @@ GEM
     blueprinter (1.1.0)
     bootsnap (1.18.4)
       msgpack (~> 1.2)
-    brakeman (6.1.2)
+    brakeman (6.2.1)
```

Comment:
Was this upgrade needed for the stamp? Otherwise we may want to remove this

Reply:
Yes it was. I tried using the order version but it wouldn't work.